.travis.yml

```diff
@@ -10,12 +10,13 @@ before_install:
   - npm i -g npm@latest
 
 script:
+  - git clone -b master https://github.com/sCrypt-Inc/compiler_dist.git compiler
   - npm test
   # integration testing with boilerplate
   - npm link
-  - git clone -b array https://github.com/scrypt-sv/boilerplate.git
+  - git clone -b master https://github.com/scrypt-sv/boilerplate.git
   - cd boilerplate
   - npm install
   - npm link scryptlib  # use this version
   #- npm ls scryptlib    # double check the latest version is used
-  - npm test
\ No newline at end of file
+  - npm test -- --ci=true
```

Comment:
No need to change

Reply:
we need to get scryptc while run test on ci. see this commit
https://github.com/sCrypt-Inc/boilerplate/commit/be138595c611f4ec0a7d7346c7c88cf601b3bf48